### PR TITLE
Update styling

### PR DESCRIPTION
**Pull request by AELSchauer**

### CODE REVIEW

File: app/controllers/donations_controller.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class DonationsController < ApplicationController
 
   def index
     @donations = Donation.all
+    @donations2 = Donation.all.each_slice(4).to_a
```
Comment (TheKaterTot) [inline app/controllers/donations_controller.rb:6]: I don't think I understand what this is doing. Is this changing the view?

**Comment by AELSchauer** [comment]
Due to many merge conflicts that were unable to be resolved in the browser, @TheKaterTot had to merge everything locally. She will submit a new PR once that local review is complete.

File: spec/features/user/user_can_add_donations_to_their_cart_spec.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      expect(current_path).to eq donations_path
+      expect(page).to have_content("You now have 1 #{@donation.title}.")
+    end
+
+    scenario "the message correctly increments for multiple items" do
+      visit donation_path(@donation)
+      click_button "Add to Cart"
+
+      expect(page).to have_content("You now have 1 #{@donation.title}.")
+      click_on "#{@donation.title}"
```
[the remark this answers is not in the document]
Comment (edilenedacruz) [reply spec/features/user/user_can_add_donations_to_their_cart_spec.rb:22]: Breaks test

File: spec/features/admin/admin_can_see_order_subtotals_spec.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe "Admin can see order subtotals" do
+  scenario "admin can see order subtotals" do
+
+    admin = create(:user, role: 1)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+    ordered = create(:order, status: "ordered")
+    paid = create_list(:order, 2, status: "paid")
```
[the remark this answers is not in the document]
Comment (edilenedacruz) [reply spec/features/admin/admin_can_see_order_subtotals_spec.rb:10]: For organization purposes.

**Pull request closed without merging**